src/templates/blogpost.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+import Layout from '../components/layout'
+import Block from '../components/block'
+import Break from '../components/break'
+import Prose from '../components/prose'
+
+
+export const BlogPostTemplate = ({
+  title,
+  body,
+  subtitle,
+  background,
+  audio,
+}) => (
+  <div>
+    <Block 
+      className={'bg-white'}
+    >
+      <Prose>
+        <div className={'pb-10'}>
+          <div className={'mb-5'}>
+            <a className={'text-blue-light text-md'} href={'/blog'}>View more articles</a>
+          </div>
+          <h1 className={'text-blue-dark text-4xl md:text-5xl leading-none font-bold mb-1'}>{title}</h1>
+          <div className={'text-xl md:text-2xl leading-tight mb-5'}>{subtitle}</div>
+          <Break />
+          <img src={background?.childImageSharp?.fluid?.src} className={'w-full mb-5'} />
```

Comment:
I think I might make this image `aria-hidden` for now, but @JQuaggles, would it be helpful to add a field for entering custom alt text for these images?

currently, we only really have the "title" and "subtitle" to use here as image alt, but that's already provided in the header and subtitle and would be redundant here.  It could be nice to allow our blog writers to describe the image selected for the post in cases where the image helps provide context or setting for the article.